src/faux-macros.lisp

```diff
@@ -51,6 +51,3 @@
 
 (define-coalton-editor-macro coalton:lisp (type vars &body lisp-expr)
   "An escape from Coalton into the Lisp world.")
-
```

Comment:
was this intended?

Reply:
Yeah. It's already a macro in `language-macros.lisp` so it doesn't need an editor macro.